Source/CarthageKit/Version.swift

```diff
@@ -106,44 +106,12 @@ public struct SemanticVersion: VersionType {
 }
 
 extension SemanticVersion: Scannable {
-	/// Attempts to parse a semantic version from a human-readable string of the
-	/// form "a.b.c" from a string scanner. It assumes the string might contain a
-	/// comment coming from a Cartfile and will ignore the comment if present
```

Comment:
Nit: grammar: `the comment, if present.`

Reply:
This comment line is now gone anyway.

Reply:
🤦‍♂️  — misread my note about the comment [here](https://github.com/Carthage/Carthage/pull/2469/files#diff-75029f8fcb86c4555bca7116130d72b6R181)

so minor… no need to do anything about that…